src/main/java/io/jenkins/plugins/audit/listeners/UserLogListener.java

```diff
@@ -93,19 +96,20 @@ protected void loggedIn(@Nonnull String username) {
      *
      * @param username name or ID of the user who logged out.
      */
-     @Override
-     protected void loggedOut(@Nonnull String username) {
-         String currentTime = Instant.now().toString();
-         Logout logout = LogEventFactory.getEvent(Logout.class);
+    @Override
+    protected void loggedOut(@Nonnull String username) {
+        String currentTime = Instant.now().toString();
+        Logout logout = LogEventFactory.getEvent(Logout.class);
 
-         RequestContext.setNodeName("master");
-         RequestContext.getRequestId();
-         RequestContext.setIpAddress(Stapler.getCurrentRequest().getRemoteAddr());
-         logout.setUserId(username);
-         logout.setTimestamp(currentTime);
-         logout.logEvent();
-         RequestContext.clear();
-     }
+        RequestContext.setNodeName("master");
+        RequestContext.getRequestId();
```

Comment:
Unclear on the need for this as you're ignoring its return value.

Reply:
the getter was there to cause the `requestId` to be put into the ThreadContext since there isn't an explicit `setRequestId()` method in the `RequestContext` class. or should i create one there ?

Reply:
The request id should be created and set by the servlet filter.

Reply:
since the `requestId` is a randomly generated UUID i'm not sure if having a setter for it is a good idea - would the servlet filter be generating its on request IDs ?

Reply:
The servlet filter would be in charge of generating the request ids, yes.